<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="1" total="6" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6137" started-at="2018-12-16T10:51:34Z" finished-at="2018-12-16T10:51:40Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6137" started-at="2018-12-16T10:51:34Z" finished-at="2018-12-16T10:51:40Z">
      <class name="com.urqa.JiraRestAssured.JiraProcess">
        <test-method status="PASS" signature="getProperty()[pri:0, instance:com.urqa.JiraRestAssured.JiraProcess@1deb50e]" name="getProperty" is-config="true" duration-ms="12" started-at="2018-12-16T16:21:34Z" finished-at="2018-12-16T16:21:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getProperty -->
        <test-method status="PASS" signature="getAuthentication()[pri:1, instance:com.urqa.JiraRestAssured.JiraProcess@1deb50e]" name="getAuthentication" duration-ms="4280" started-at="2018-12-16T16:21:34Z" finished-at="2018-12-16T16:21:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAuthentication -->
        <test-method status="PASS" signature="createProject()[pri:0, instance:com.urqa.JiraRestAssured.JiraProcess@1deb50e]" name="createProject" duration-ms="1527" started-at="2018-12-16T16:21:38Z" depends-on-methods="com.urqa.JiraRestAssured.JiraProcess.getAuthentication" finished-at="2018-12-16T16:21:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createProject -->
        <test-method status="PASS" signature="getAllProject()[pri:0, instance:com.urqa.JiraRestAssured.JiraProcess@1deb50e]" name="getAllProject" duration-ms="211" started-at="2018-12-16T16:21:39Z" depends-on-methods="com.urqa.JiraRestAssured.JiraProcess.createProject" finished-at="2018-12-16T16:21:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllProject -->
        <test-method status="FAIL" signature="updateProject()[pri:0, instance:com.urqa.JiraRestAssured.JiraProcess@1deb50e]" name="updateProject" duration-ms="43" started-at="2018-12-16T16:21:40Z" depends-on-methods="com.urqa.JiraRestAssured.JiraProcess.createProject" finished-at="2018-12-16T16:21:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <201> but was <405>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <201> but was <405>.

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:238)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:483)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:655)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:169)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:123)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:131)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:119)
	at com.urqa.JiraRestAssured.JiraProcess.updateProject(JiraProcess.java:167)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateProject -->
        <test-method status="SKIP" signature="deleteProject()[pri:0, instance:com.urqa.JiraRestAssured.JiraProcess@1deb50e]" name="deleteProject" duration-ms="0" started-at="2018-12-16T16:21:40Z" depends-on-methods="com.urqa.JiraRestAssured.JiraProcess.updateProject" finished-at="2018-12-16T16:21:40Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method JiraProcess.deleteProject()[pri:0, instance:com.urqa.JiraRestAssured.JiraProcess@1deb50e] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method JiraProcess.deleteProject()[pri:0, instance:com.urqa.JiraRestAssured.JiraProcess@1deb50e] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteProject -->
      </class> <!-- com.urqa.JiraRestAssured.JiraProcess -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
